src/cookieconsent.js

```diff
@@ -1363,14 +1363,12 @@
         if (nextService) {
           this.runService(nextService, this.runNextServiceOnError.bind(this));
         } else {
-          this.completeService.call(
-            this,
-            this.callbackError,
-            new Error('All services failed')
-          );
+          this.currentServiceIndex = -1;
+          this.callbackError(new Error('All location services failed'));
         }
       } else {
-        this.completeService.call(this, this.callbackComplete, data);
+        this.currentServiceIndex = -1;
+        this.callbackComplete(data);
```

Comment:
The code that previously called this checked the function existed before calling. Looking at where `this.callbackComplete` and `this. callbackError` are defined it's possible that the function returns before they're set. Without reading the whole file I can't tell if it's possible for these to be called without them being defined?